src/models/User.model.js

```diff
@@ -1,9 +1,26 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const User = sequelize.define(
-  // your code goes here
+  'User',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    name: {
+      type: DataTypes.STRING,
```

Comment:
Consider specifying a maximum length for the `name` field to ensure data consistency and prevent excessively long entries.